config/core.entity_form_display.node.training.default.yml

```diff
@@ -185,7 +185,7 @@ content:
         'source:field_user_posting_rights': 0
     third_party_settings: {  }
   field_theme:
-    type: options_buttons
+    type: reliefweb_options
```

Comment:
I don't see the code for the widget in the PR but if that's not already the case, please have a look at the https://github.com/UN-OCHA/rwint9-site/blob/develop/html/modules/custom/reliefweb_fields/src/Plugin/Field/FieldWidget/ReliefWebEntityReferenceSelect.php widget which adds widget options to add fields as data attributes to select option elements.

This widget is for `select` but could be adapted easily enough for `checkboxes/radios`.

It's used for example for the disaster field on the report form to add the disaster status as a data attribute which is then used to add a colored border to the suggestions.

Reply:
Missing file added